package.json

```diff
@@ -25,24 +25,24 @@
     "clean:modules": "rimraf node_modules"
   },
   "engines": {
-    "node": ">=16.0.0"
+    "node": ">=20.0.0"
   },
   "dependencies": {
-    "@actions/core": "^1.6.0",
-    "@actions/github": "^5.0.0",
-    "@octokit/action": "^3.18.0",
-    "axios": "^0.25.0",
+    "@actions/core": "^1.10.1",
+    "@actions/github": "^6.0.0",
+    "@octokit/action": "^6.0.7",
+    "axios": "^1.6.7",
     "is-plain-object": "^5.0.0",
     "lcov-parse": "^1.0.0"
   },
   "devDependencies": {
-    "@types/jest": "^27.4.0",
-    "eslint": "^7.17.0",
-    "eslint-config-neo": "^0.6.3",
-    "jest": "^27.5.0",
-    "prettier": "^2.5.1",
-    "rimraf": "^3.0.2",
-    "ts-jest": "^27.1.3",
-    "typescript": "^4.5.5"
+    "@types/jest": "^29.5.12",
```

Comment:
check if these support running -- --coverage the same 

the action relies on the --coverage file being generated and the build looks like it cant find it. which makes me think its not being generated. 

if you pull this locally and run `npm run test -- --coverage` and an lcov.info file is generated than this is not the problem
if it is not generated, then this is likely the problem

Reply:
It passed locally :(

Reply:
using node 20?

Reply:
Yup! everything works locally

Reply:
on this version of this branch